cmd/argocd/commands/cluster.go

```diff
@@ -276,31 +277,56 @@ argocd cluster rm cluster-name`,
 			}
 			conn, clusterIf := headless.NewClientOrDie(clientOpts, c).NewClusterClientOrDie()
 			defer io.Close(conn)
+			var numOfClusters = len(args)
+			var isConfirmAll bool = false
 
 			for _, clusterSelector := range args {
 				clusterQuery := getQueryBySelector(clusterSelector)
+				var lowercaseAnswer string
+				if !noPrompt {
+					if numOfClusters == 1 {
+						lowercaseAnswer = cli.AskToProceedS("Are you sure you want to remove '" + clusterSelector + "' and all its resources? [y/n] ")
```

Comment:
How about "Are you sure you want to remove 'cluster'? Any Apps deploying to this cluster will go to health status Unknown."